<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hotels in clouds</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="registering">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/Image/kua4.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(122, 56, 3)
}
QPushButton:hover
{
border-image: url(:/Image/kua5.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(244, 234, 41)
}
QPushButton:pressed
{
border-image: url(:/Image/kua3.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(153, 94, 30)
}</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_phonenum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(94, 213, 238);</string>
    </property>
    <property name="text">
     <string>用户名（手机）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(94, 213, 238);</string>
    </property>
    <property name="text">
     <string>码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_class">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(94, 213, 238);</string>
    </property>
    <property name="text">
     <string>用户类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;方正清刻本悦宋简体&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>顾客</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>管理员</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>酒店</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>平台</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phnum">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_psw">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/Image/kuang4.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(213, 108, 15)
}
QPushButton:hover
{
border-image: url(:/Image/kuang6.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(244, 234, 41)
}
QPushButton:pressed
{
border-image: url(:/Image/kuang7.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(153, 94, 30)
}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/Image/kuang4.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(213, 108, 15)
}
QPushButton:hover
{
border-image: url(:/Image/kuang6.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(244, 234, 41)
}
QPushButton:pressed
{
border-image: url(:/Image/kuang7.png);
font: 100 11pt &quot;方正清刻本悦宋简体&quot;;
color:rgb(153, 94, 30)
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(13, 158, 189);
font: 75 12pt &quot;方正清刻本悦宋简体&quot;;</string>
    </property>
    <property name="text">
     <string>没有账号？</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(10, 74, 88);
font: 75 12pt &quot;方正清刻本悦宋简体&quot;;
</string>
    </property>
    <property name="text">
     <string>密</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
